{"name": "Безуспешное вычисление остатка от деления на 0", "status": "failed", "statusDetails": {"message": "AssertionError: expected internal status code: 1\nactual internal status code: 8", "trace": "self = <tests.calculator_api.test_remainder.TestRemainder object at 0x00000276A4352F00>\ncalculator_steps = <steps.calculator_api.calculator_steps.CalculatorSteps object at 0x00000276A5480C20>, operands = IntegerOperands(x=0, y=0)\n\n    @allure.title('Безуспешное вычисление остатка от деления на 0')\n    @pytest.mark.parametrize(\"operands\",\n                             [IntegerOperands(x=5, y=0),\n                              IntegerOperands(x=0, y=0)],\n                             ids=idfn)\n    def test_unsuccessful_remainder_by_zero(self, calculator_steps, operands):\n        response = calculator_steps.do_remainder(operands)\n    \n        # assert_http_status_code(response=response, expected_code=422)\n        assert_json_header(response=response)\n    \n        error = ServerSchemasSteps.get_server_error(response)\n>       assert_calculator_calculation_error(calculation_error=error)\n\ntests\\calculator_api\\test_remainder.py:134: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nassertions\\calculator_assertions.py:68: in assert_calculator_calculation_error\n    assert_internal_status_code(calculation_error, 1)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\ndata = ServerError(status_code=8, status_message='Ошибка вычисления'), expected_status_code = 1\n\n    def assert_internal_status_code(data: BaseModel, expected_status_code):\n>       assert data.status_code == expected_status_code, \\\n            f'expected internal status code: {expected_status_code}\\nactual internal status code: {data.status_code}\\n'\nE       AssertionError: expected internal status code: 1\nE       actual internal status code: 8\n\nassertions\\base_assertions.py:45: AssertionError"}, "steps": [{"name": "Посчитать остаток от деления чисел", "status": "passed", "steps": [{"name": "Остаток от деления x на y", "status": "passed", "steps": [{"name": "Отправка \"'POST'\" запроса на \"'/remainder'\"", "status": "passed", "parameters": [{"name": "method", "value": "'POST'"}, {"name": "url", "value": "'/remainder'"}, {"name": "content", "value": "None"}, {"name": "data", "value": "None"}, {"name": "files", "value": "None"}, {"name": "json", "value": "{'x': 0, 'y': 0}"}, {"name": "params", "value": "None"}, {"name": "headers", "value": "None"}, {"name": "cookies", "value": "None"}, {"name": "auth", "value": "<httpx._client.UseClientDefault object at 0x00000276A3624C20>"}, {"name": "follow_redirects", "value": "<httpx._client.UseClientDefault object at 0x00000276A3624C20>"}, {"name": "timeout", "value": "<httpx._client.UseClientDefault object at 0x00000276A3624C20>"}, {"name": "extensions", "value": "None"}], "start": 1723569765742, "stop": 1723569765758}], "parameters": [{"name": "operands_payload", "value": "{'x': 0, 'y': 0}"}], "start": 1723569765742, "stop": 1723569765758}], "parameters": [{"name": "operands", "value": "IntegerOperands(x=0, y=0)"}], "start": 1723569765742, "stop": 1723569765758}, {"name": "Получить ошибку сервера из ответа", "status": "passed", "parameters": [{"name": "response", "value": "<Response [200 OK]>"}], "start": 1723569765758, "stop": 1723569765758}], "parameters": [{"name": "operands", "value": "IntegerOperands(x=0, y=0)"}], "start": 1723569765742, "stop": 1723569765758, "uuid": "cbe81647-4505-4023-9007-18407799b11b", "historyId": "26a4ed620823d860165ae5b9d81561e7", "testCaseId": "0f7c383d12375fe3229110597292629f", "fullName": "tests.calculator_api.test_remainder.TestRemainder#test_unsuccessful_remainder_by_zero", "labels": [{"name": "story", "value": "Вычисление остатка от деления"}, {"name": "epic", "value": "Калькулятор"}, {"name": "feature", "value": "Вычисления калькулятора"}, {"name": "parentSuite", "value": "tests.calculator_api"}, {"name": "suite", "value": "test_remainder"}, {"name": "subSuite", "value": "TestRemainder"}, {"name": "host", "value": "DESKTOP-QQNQKOJ"}, {"name": "thread", "value": "11904-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.calculator_api.test_remainder"}]}